stacks/templates/front-page.html

```diff
@@ -0,0 +1 @@
+<!-- wp:pattern {"slug":"stacks/home"} /-->
```

Comment:
Why front-page.html rather than home.html?

Reply:
Thinking was this pattern or type of experience is better suited to single page sites, which front-page is ideal for.